homeassistant/components/cover/zwave.py

```diff
@@ -110,24 +123,44 @@ class ZwaveGarageDoor(zwave.ZWaveDeviceEntity, CoverDevice):
     def __init__(self, values):
         """Initialize the zwave garage door."""
         ZWaveDeviceEntity.__init__(self, values, DOMAIN)
+        self._cover_state_list = None
+        self._cover_state = None
         self.update_properties()
 
     def update_properties(self):
         """Handle data changes for node values."""
-        self._state = self.values.primary.data
+        self._cover_state = ZWAVE_STATES.get(self.values.primary.data)
```

Comment:
Might need to keep the ZWAVE_STATES as STATE_OPEN = "open" and the ZwaveLib is looking for "opened" not "open".